public/assets/gameRules.json

```diff
@@ -0,0 +1,10 @@
+{
+  "gameDescription": "The players take on the part of dwarves. Either they are gold-diggers, working their tunnels ever deeper into the mountain in search of treasures, or they are saboteurs trying to put obstacles into the diggers’ path. The members of each group should support each other, even though often they can only guess who is what. If the gold-diggers manage to create a path to the treasure, they are duly rewarded with gold, while the saboteurs have to leave with coal in their pockets. However, should the gold-diggers fail, the saboteurs reap the reward. Only when the gold is found are the roles revealed.",
```

Comment:
Great wordplay on "gold-diggers", for clarity i would write "Either they are miners (gold-diggers), working their ..."